tests/e2e/multicluster/multicluster_primaryremote_test.go

```diff
@@ -302,6 +303,82 @@ spec:
 					})
 				})
 
+				When("A revision is no longer in use", func() {
```

Comment:
nit (as its a convention)
```suggestion
				When("a revision is no longer in use", func() {
```